game/forms_avatar.py

```
@@ -0,0 +1,45 @@
+                file_name=file_name,
+                content_type=content_type,
+                size_bytes=size_bytes,
+                file_bytes=file_bytes,
+            )
+        except ValueError as exc:
+            raise forms.ValidationError(str(exc))
+
+        # Ensure decodable/processable.
+        try:
+            process_avatar_image(file_bytes)
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)